app/src/main/java/org/mifos/selfserviceapp/data/FundTransfer/FundTransferTemplateResponse.java

```diff
@@ -0,0 +1,30 @@
+package org.mifos.selfserviceapp.data.FundTransfer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Vishwajeet
+ * @since 23/8/16.
+ */
+
+public class FundTransferTemplateResponse {
+    private List<FundTransferTemplate> fromAccountOptions = new ArrayList<>();
+    private List<FundTransferTemplate> toAccountOptions = new ArrayList<>();
+
+    public List<FundTransferTemplate> getFromAccountOptions() {
+        return fromAccountOptions;
+    }
+
+    public void setFromAccountOptions(List<FundTransferTemplate> fromAccountOptions) {
```

Comment:
nit: you may not need the set methods for these. Since these are usually populated by retrofit, you can just have the get methods here.